lib/elixir/lib/gen_server/loop.ex

```diff
@@ -0,0 +1,392 @@
+defmodule GenServer.Loop do
```

Comment:
Should we merge this inside `GenServer`? Unless you have a strong reason to keep the loop out otherwise I would prefer to keep it on `GenServer` for simplicity and stacktrace purposes.